app/models/MessageModel.php

```diff
@@ -50,6 +55,37 @@ function exists($id) {
       return ($this->get($id) !== NULL);
     }
 
+    function getNumUnread($studentOrRecruiterId) {
+      if (StudentModel::exists($studentOrRecruiterId)) {
+        return StudentModel::getNumUnread(new MongoId($studentOrRecruiterId));
+      } else if (RecruiterModel::IDexists($studentOrRecruiterId)) {
+        return RecruiterModel::getNumUnread(new MongoId($studentOrRecruiterId));
+      } else {
+        // This shouldn't happen!
```

Comment:
This can be done with an invariant call. This way if this ever reaches here, "Something went wrong!" would show up and we would get an error message.